apps/transport/lib/db/contact.ex

```diff
@@ -163,7 +163,7 @@ defmodule DB.Contact do
   def organization_name(orgs) do
     certified_orgs =
       orgs
-      |> Enum.filter(fn %DB.Organization{badges: badges} -> Enum.any?(badges, &match?(&1, %{"kind" => "certified"})) end)
+      |> Enum.filter(fn %DB.Organization{badges: badges} -> Enum.any?(badges, &match?(%{"kind" => "certified"}, &1)) end)
```

Comment:
```suggestion
    certified_orgs = Enum.filter(orgs, fn %DB.Organization{badges: badges} -> %{"kind" => "certified"} in badges end)
```

pour simplifier. Il y a des doctests sur cette méthode

Reply:
J’ai fait un commit avec ta suggestion (je ne pouvais pas l’appliquer depuis Github).